tests/testthat/test-backend-.R

```diff
@@ -103,9 +103,9 @@ test_that("lead and lag translate n to integers", {
 
 test_that("can translate case insensitive like", {
```

Comment:
```suggestion
test_that("can't translate case insensitive like", {
```
?

Could you please also add a test for case sensitive like? Or maybe come up with a more encompassing test name and test both sides of this function in the same test?

Reply:
I have updated the tests. I'm not sure how to add more tests for case sensitive like here beyond checking for the expected errors. Where ILIKE is being used, for example for postgres, there are tests of it in the backend specific tests